core/src/edu/cornell/gdiac/main/PlayerModel.java

```diff
@@ -371,6 +371,57 @@ public boolean isFacingRight() {
     return faceRight;
   }
 
+  /**
+   * Returns the frozen texture
+   */
+  public TextureRegion getFrozenTexture() {
+    return frozenTexture;
+  }
+
+  /**
+   * Sets the texture for frozen player
+   *
+   * @param texture the texture to set frozenTexture to
+   */
+  public void setFrozenTexture(TextureRegion texture) {
+    frozenTexture = texture;
+  }
+
+  /**
+   * Sets sensor shape as box
+   */
+  public void setSensorShape(float sensorSizeX, float sensorSizeY, Vector2 sensorCenter,
+      float angle) {
+    sensorShape = new PolygonShape();
+    sensorShape.setAsBox(sensorSizeX, sensorSizeY, sensorCenter, angle);
+  }
+
+  /**
+   * Sets sensor color
+   *
+   * @param color the color to set the sensor color to
+   */
+  public void setSensorColor(Color color){
+    sensorColor = color;
+  }
+
+  /**
+   * Sets fall multiplier
+   *
+   * @param value the value to set the fall multiplier to
+   */
+  public void setFallMultiplier(float value){
+    fallMultiplier = value;
+  }
+
+  /**
+   * Sets low jump multiplier
+   *
+   * @param value the value to set the low jump multiplier
+   */
+  public void setLowJumpMultiplier(float value){
+    lowJumpMultiplier = value;
+  }
```

Comment:
Once again none of these method should exist since we want to initialize them in the initialize method within the player model itself.